.gitignore

```diff
@@ -1,3 +1,2 @@
 .idea/
 *.gem
-Gemfile.lock
```

Comment:
http://yehudakatz.com/2010/12/16/clarifying-the-roles-of-the-gemspec-and-gemfile/